packages/kbn-unified-data-table/src/utils/get_render_cell_value.tsx

```diff
@@ -6,31 +6,24 @@
  * Side Public License, v 1.
  */
 
-import React, { Fragment, useContext, useEffect } from 'react';
-import classnames from 'classnames';
+import React, { useContext, useEffect } from 'react';
 import { i18n } from '@kbn/i18n';
 import { euiLightVars as themeLight, euiDarkVars as themeDark } from '@kbn/ui-theme';
 import type { DataView, DataViewField } from '@kbn/data-views-plugin/public';
 import {
   EuiDataGridCellValueElementProps,
-  EuiDescriptionList,
-  EuiDescriptionListTitle,
-  EuiDescriptionListDescription,
   EuiButtonIcon,
   EuiFlexGroup,
   EuiFlexItem,
 } from '@elastic/eui';
 import type { FieldFormatsStart } from '@kbn/field-formats-plugin/public';
-import type {
-  DataTableRecord,
-  EsHitRecord,
-  ShouldShowFieldInTableHandler,
-  FormattedHit,
-} from '@kbn/discover-utils/types';
-import { formatFieldValue, formatHit } from '@kbn/discover-utils';
+import type { DataTableRecord, ShouldShowFieldInTableHandler } from '@kbn/discover-utils/types';
+import { formatFieldValue } from '@kbn/discover-utils';
 import { UnifiedDataTableContext } from '../table_context';
-import { defaultMonacoEditorWidth } from '../constants';
-import JsonCodeEditor from '../components/json_code_editor/json_code_editor';
+import { CustomCellRenderer } from '../..';
```

Comment:
Would be better to import from the types file only so it does not depend on the other included exports.

Reply:
If i understand correctly, you mean explicit from the file name -
```import type { CustomCellRenderer } from '../types';```